src/js/models/markup.js

```diff
@@ -10,27 +11,36 @@ export const VALID_MARKUP_TAGNAMES = [
   'li'
 ].map(normalizeTagName);
 
+export const VALID_ATTRIBUTES = [
```

Comment:
Yeah, I think so — and in fact, VALID_ATTRIBUTES should most likely be contextual. `href` isn't valid on a `<em>` tag, for instance. I'll move the attribute-filtering (as is) into Markup directly, for now.